src/components/AboutDialog/AboutDialog.tsx

```diff
@@ -25,7 +24,7 @@ function AboutDialog({ open, onClose }: PropsWithChildren<AboutDialogProps>) {
       <DialogContent>
         <DialogContentText>Browser supported: {String(Video.isSupported)}</DialogContentText>
         <DialogContentText>SDK Version: {Video.version}</DialogContentText>
-        <DialogContentText>App Version: {appVersion}</DialogContentText>
+        <DialogContentText>App Version: {process.env.REACT_APP_VERSION}</DialogContentText>
```

Comment:
Here I wonder what we should have when we run `npm start`?
Right now it just looks like `App Version:` with nothing after it. Should we remove it altogether? Or say "See package.json"? Or should we populate REACT_APP_VERSION when we run `npm start`? What do you think?

Reply:
I added it to `npm start`. I thought about it and figured it is an easy way to check the app version so why not. It was also an easy addition 😄 . On the other hand, I suppose getting rid of it altogether isn't a bad idea either..